go-backend/service/user_http_test.go

```diff
@@ -67,9 +69,13 @@ func (suite *UsersHandlerTestSuite) TestListUsersWhenDBFailure() {
 	suite.dbMock.AssertExpectations(suite.T())
 }
 
-func makeHTTPCall(method, path, body string, handlerFunc http.HandlerFunc) (recorder *httptest.ResponseRecorder) {
+// Use path variable for configure our test router with given handler and requestURL variable for serve the http request
```

Comment:
Change the comment to:

path: is used to configure router path (eg: /users/{user_id})
requestURL: current request path (eg: /users/1)